src/data/contributors/Kalyani.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "Kalyani Kolhe",
+    "skills" : ["C", "C++", "HTML", "JS"],
+    "cause" : "I code to develop my skills",
```

Comment:
`code for` is already the part of phrase, You should only enter the `cause`.
![image](https://user-images.githubusercontent.com/48255244/90138030-06f77380-dd94-11ea-8640-b822652a3f7a.png)

Reply:
Yes sure, I got it.
I'll resolve it and again pull a request